### ISSUE TITLE
"iob add name --host otherhost" should foreward command to correct host

### ISSUE THREAD

**Issue by Apollon77** [ioBroker/ioBroker.js-controller#1716]
We currently already detect that the host should be a different one but then normal installation is done.

https://github.com/ioBroker/ioBroker.js-controller/blob/9079df08acfa9720e1e1747d3476950e7ec51ca5/packages/controller/lib/setup/setupInstall.js#L750

The CLI should forward the command to the relevant host to be executed there

**Comment by Apollon77** [comment]
Forwarding via "sendTo" will be a bit fragile ... in fact npm install can take looooooooong time, so how long to wait? Thats why the question if it is not a pure admin issue ...

**Comment by foxriver76** [comment]
> And yes in controller we should not install it locally at all, but use data (from repo?) to create a dummy instance object with the target host ... Then first adapter start does the rest (aka install if not there)

Good idea, not sure if repo has relevant data to create the instanceObj.. maybe do like now and del adapter afterwards from the executing host if host flag was remote host.. much work for little outcome, so maybe we can get instanceObj information elsewhere.

**Comment by Apollon77** [comment]
When I remember teh repo has the io-package with shortened news (but this is removed anyway for instance object) ... would be a try :-)
